spk/sonarr/src/service-setup.sh

```diff
@@ -90,17 +82,21 @@ service_postupgrade ()
     . ${CONFIG_DIR}/KEEP_VAR
     if [ "$KEEP_CUR" == "yes" ]; then
         echo "Restoring Sonarr version from before upgrade" >> ${INST_LOG}
-        rm -fr ${SYNOPKG_PKGDEST}/share >> $INST_LOG 2>&1
-        mv ${INST_VAR}/share ${SYNOPKG_PKGDEST}/ >> $INST_LOG 2>&1
+        rm -fr ${SYNOPKG_PKGDEST}/share >> ${INST_LOG} 2>&1
+        mv ${INST_VAR}/share ${SYNOPKG_PKGDEST}/ >> ${INST_LOG} 2>&1
         set_unix_permissions "${SYNOPKG_PKGDEST}/share"
     fi
 
     set_unix_permissions "${CONFIG_DIR}"
 
     # If backup was created before new-style packages
     # new updates/backups will fail due to permissions (see #3185)
-    set_unix_permissions "/tmp/nzbdrone_backup"
-    set_unix_permissions "/tmp/nzbdrone_update"
+    if [ -d "/tmp/nzbdrone_backup" ] || [ -d "/tmp/nzbdrone_update" ] || [ -d "/tmp/sonarr_backup" ] || [ -d "/tmp/sonarr_update" ]; then
+        set_unix_permissions "/tmp/nzbdrone_backup"
+        set_unix_permissions "/tmp/nzbdrone_update"
+        set_unix_permissions "/tmp/sonarr_backup"
+        set_unix_permissions "/tmp/sonarr_update"
```

Comment:
Are these still needed? I don't see anything being created there during upgrades. FYI these paths won't work in DSM7

Reply:
It might be wise to leave these until the next update to ensure there are no permissions errors that cause any problems during the upgrade. I know that these were added to most packages previously to resolve a permissions upgrade error...

Reply:
Cool, thanks for clarifying.